sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/RsaKeyCryptographyClient.java

```diff
@@ -53,20 +53,20 @@ private KeyPair getKeyPair(JsonWebKey key) {
     }
 
     @Override
-    Mono<EncryptResult> encryptAsync(EncryptOptions encryptOptions, Context context, JsonWebKey jsonWebKey) {
-        Objects.requireNonNull(encryptOptions, "'encryptOptions' cannot be null.");
-        Objects.requireNonNull(encryptOptions.getAlgorithm(), "Encryption algorithm cannot be null.");
-        Objects.requireNonNull(encryptOptions.getPlainText(), "Plain text content to be encrypted cannot be null.");
+    Mono<EncryptResult> encryptAsync(EncryptParameters encryptParameters, Context context, JsonWebKey jsonWebKey) {
+        Objects.requireNonNull(encryptParameters, "'encryptOptions' cannot be null.");
+        Objects.requireNonNull(encryptParameters.getAlgorithm(), "Encryption algorithm cannot be null.");
+        Objects.requireNonNull(encryptParameters.getPlainText(), "Plain text content to be encrypted cannot be null.");
```

Comment:
Just "plaintext". Same with "ciphertext" everywhere.

Reply:
Please refer to [my comment](https://github.com/Azure/azure-sdk-for-java/pull/20357#discussion_r607426548) above.